src/adapter/Cargo.toml

```diff
@@ -27,7 +27,7 @@ hex = "0.4.3"
 http = "0.2.8"
 itertools = "0.10.5"
 once_cell = "1.16.0"
-launchdarkly-server-sdk = { version = "1.0.0", default_features = false, features = [
+launchdarkly-server-sdk = { version = "1.0.0", default-features = false, features = [
```

Comment:
👀 This enabling more features than needed before?

Reply:
No it was still disabling default features, they just deprecated the snake casing format in favor of kebab casing